pacemaker_metrics.go

```diff
@@ -237,6 +243,16 @@ func (c *pacemakerCollector) recordFailCountMetrics(pacemakerStatus pacemakerSta
 	}
 }
 
+func (c *pacemakerCollector) recordResourceAgentsChanges(pacemakerStatus pacemakerStatus, ch chan<- prometheus.Metric) {
+	t, err := time.Parse(time.ANSIC, pacemakerStatus.Summary.LastChange.Time)
+	if err != nil {
+		log.Warnln(err)
+		return
+	}
+	// is the resource have changed we set a different timeout from pacemaker
```

Comment:
This comment is a bit unclear :stuck_out_tongue_winking_eye: 

```suggestion
	// The time of the last change is used to set the metric timestamp. The value is always 1 and is not relevant.
```